examples/ws/JSON/src/json/JSONParser.java

```diff
@@ -0,0 +1,18 @@
+
+package json;
+
+/**
+ * @Desc The main class for parsing a JSON via remote or local.
+ * @author jash, haotao
+ */
+public class JSONParser {
+    
+    public static void main(String [] args){
+//        JSONParserLocal ojsonLocal = new JSONParserLocal();
+//        ojsonLocal.localJsonParser();
```

Comment:
Make them selectable by an option instead of commented out. Like ``if(argv.length > 0  && argv[0].equals("--local")) ....``